docs/content/development/extensions-contrib/graphite.md

```diff
@@ -9,7 +9,7 @@ To use this extension, make sure to [include](../../operations/including-extensi
 ## Introduction
 
 This extension emits druid metrics to a graphite carbon server.
-Events are sent after been [pickled](http://graphite.readthedocs.org/en/latest/feeding-carbon.html#the-pickle-protocol); the size of the batch is configurable. 
+Metrics can be sent by using [plaintext](http://graphite.readthedocs.io/en/latest/feeding-carbon.html#the-plaintext-protocol) or [pickle](http://graphite.readthedocs.io/en/latest/feeding-carbon.html#the-pickle-protocol) protocol; the size of the batch is configurable(only pickle protocol).
```

Comment:
nit space before `(`. Can we add explanation about the difference between plain text and pickled one, like the performance implication ? i think we need to state that in the plaintext is sending an event per connection which can be very inefficient.